src/domains/citizen-auth-common/06_data.tf

```diff
@@ -33,3 +33,18 @@ data "azurerm_linux_function_app" "lollipop_function" {
   resource_group_name = data.azurerm_resource_group.lollipop_function_rg.name
 }
 #######################
+
+data "azurerm_resource_group" "data_rg_itn" {
+  name = "${local.project_itn}-data-rg-01"
+}
```

Comment:
Since the data is only used to get the name of the resource group and its input is actually that name, I think we can get rid of it

```suggestion
```